api/dp-openapi.yaml

```diff
@@ -591,6 +591,354 @@ paths:
         default:
           $ref: '#/components/responses/UnexpectedError'
 
+  /projects/badpagedata:
```

Comment:
We should think of a more generic way to handle string translation than usecase-specific URLs. Others have solved this problem so lets do some research on how they've done it rather than reinventing our own.

Reply:
Yes, the way we are handling translated strings in javascript now is somewhat inconvenient.

Reply:
Something like https://stackoverflow.com/questions/431672/how-to-translate-strings-in-js might be possible. I'll try something out.